charts/backingservices/README.md

```diff
@@ -31,18 +31,19 @@ If an externally managed elasticsearch cluster is being used, make sure the serv
 You may enable the component of [Elasticsearch](https://github.com/helm/charts/tree/master/stable/elasticsearch/values.yaml) in the backingservices by configuring the 'srs.srsStorage' section in values.yaml file to deploy ElasticSearch cluster automatically. For more configuration options available for each of the components, see their Helm Charts.
 
 Note: Pega does **not** actively update the elasticsearch dependency in `requirements.yaml`. To leverage SRS, you should:
-* Update the elasticsearch `version` value, or
+
+* Update the elasticsearch `version` value (the version value in `requirements.yaml` should be same as imageTag in [values.yaml](./values.yaml)), or
```

Comment:
Please refer to the way the pega readme distinguishes between values and parameters? I believe we are asking clients to update parameters here so we can provide precise guidance ;-):
* To use the internally-provided Elasticsearch service in the SRS cluster, use the default `srs.enabled.true`parameter and set the Elasticsearch version by updating the `elasticsearch.imageTag` parameter in the [values.yaml](./values.yaml) to match the `dependencies.version` parameter in the [requirements.yaml](./requirements.yaml).